### ISSUE TITLE
Cannot import Yahoo finance 'Stock' in the resource

### ISSUE THREAD

**Issue by sudeen** [TechPrimers/stock-price-viewer-microservices-part1#1]
In the resource part List<Stock>.. Stock cannot be imported.
I tried using different version of yahoo finance api but could not get the solution.

**Comment by ramazansakin** [comment]
Yes, probably the dependency is deprecated, I also tried and then i removed that dependency and refactored the code block. You can check on the repo
https://github.com/ramazansakin/stock-price-viewer-microservices-part1
